cookiecutter.json

```diff
@@ -1,12 +1,13 @@
 {
-    "full_name": "Karl Hobley",
-    "email": "karl@kaed.uk",
-    "github_username": "kaedroho",
+    " ": "]\nCookiecutter Wagtail Package - Use a project name without `Wagtail` as the prefix, this wil be added for you.\n[Please press enter to continue",
```

Comment:
Typo: wil

Reply:
thanks for that, fixed up & rebased now.